checklists/alz_checklist.en.json

```diff
@@ -218,13 +218,13 @@
         {
             "category": "Identity and Access Management",
             "subcategory": "Identity",
-            "text": "Enforce multi-factor authentication for any user with rights to the Azure environments.",
+            "text": "Starting in 2024, multifactor authentication (MFA) will be mandatory for all Azure sign-in attempts. Plan for and enforce multi-factor authentication for any user with rights to the Azure environments.",
```

Comment:
Can you explain why this change is needed?  It moves the recommendation text itself further into the field and makes the expected action.

Reply:
@alisakina99 we can leave this one as-is, it relates more to general MFA hygiene. 

The update in B03.14 will be sufficient to raise awareness around the changes starting in Oct 2024 around mandatory MFA